File: once_upon_a_time.py
class FizzBuzz:
    def __init__(self, start, end):
        self.LIST = [
            'FizzBuzz',
            0,
            0,
            'Fizz',
            0,
            'Buzz',
            'Fizz',
            0,
            0,
            'Fizz',
            'Buzz',
            0,
            'Fizz',
            0,
            0
        ]

        self.start = start
        self.end = end

    def run(self):
        fizzbuzz = ''
        for i in range(self.start, self.end + 1):
            fizzbuzz += '{} {}\n'.format(i, self.LIST[i % 15])

        print(fizzbuzz, end='')

if __name__ == '__main__':
    start = int(input('Start: '))
    end   = int(input('End: '))
    
    fb = FizzBuzz(start, end)
    fb.run()

File: once_upon_a_time.cpp
#include <stdio.h>
#include <string>

class FizzBuzz {
private:
    int start, end;
    const std::string LIST[15] = {
        "FizzBuzz",
        "0",
        "0",
        "Fizz",
        "0",
        "Buzz",
        "Fizz",
        "0",
        "0",
        "Fizz",
        "Buzz",
        "0",
        "Fizz",
        "0",
        "0"
    };
public:
    FizzBuzz(int start, int end) {
        this->start = start;
        this->end   = end;
    }

    void run() {
        for (int i = start; i <= end; i++) {
            printf("%d %s\n", i, LIST[i % 15]);
        }
    }
};

int main() {
    int start, end;
    scanf("%d", &start);
    scanf("%d", &end);

    FizzBuzz fb(start, end);
    fb.run();
}